.changelog/get-valid-sections.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script will output all of the valid changelog section options.
+# It extracts this list from the comment at the top of the CHANGELOG.md file.
+# Any double-quoted string at the start of a line in that comment will be treated as a valid section header.
+# Those strings are lower-cased and spaces turned to dashes.
+# E.g the line `"Bug Fixes" for any bug fixes.` will result in a valid section of "bug-fixes".
+# It's assumed that they're listed in the order that they should appear as sections.
+
+# Assume that this script is in the {repo_root}/.changelog/ dir and that the CHANGELOG.md file is directly in {repo_root}.
+where_i_am="$( cd "$( dirname "${BASH_SOURCE:-$0}" )"; pwd -P )"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)